components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt.ui/src/main/resources/web/identity-claim-mgt/update-local-claim.jsp

```diff
@@ -460,30 +460,26 @@
                     var duplicatedClaimUris = [];
                     var attributeArr = [];
 
-                    var attributeAddTable = document.getElementById('attributeAddTable').tBodies[0];
-                    var attributeAddTableRowCount = attributeAddTable.rows.length;
-
                     if (attributeAddTableRowCount > 0) {
-
-                        for (var i = 0; i < attributeAddTableRowCount; i++) {
-                            var row = attributeAddTable.rows[i];
-                            var mappedAttributeValue = row.getElementsByTagName("input")[0].value;
-                            var userstoreElem = document.getElementById("userstore_" + i);
-                            var userstoreValue = userstoreElem.options[userstoreElem.selectedIndex].value;
+                        var tRows = document.querySelectorAll('table#attributeAddTable > tbody > tr');
+                        tRows.forEach(function (elem, index) {
```

Comment:
possible NPE for tRows ?

Reply:
The Document method querySelectorAll() returns all the Elements within the document that matches the specified selector, or group of selectors. If no matches are found, null is returned. If 'table#attributeAddTable > tbody > tr' is not existing only it will return null. But these html elements are already exisiting in the jsp page. So NPE is not possible. https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector